apps/portal/components/pagesComponents/_installScreen/steps/InstallFinished.tsx

```
@@ -26,20 +29,18 @@ const InstallFinished: React.FC<IStepProps> = () => {
-            start using this tool.
-          </p>
-          <p>Or click the button below to make a screenshot right now!</p>
+          <p>{t('page.install.completedGuide')}</p>
+          <p>{t('page.install.clickRightSideExtension')}</p>
+          <p>{t('page.install.belowScreenShotButton')}</p>
+
           <div className="tw-shadow-lg tw-rounded-xl tw-overflow-hidden tw-h-60">
             <img
               className="tw-w-full"
               src="/sign/setup_finished_screen_guide.png"
             />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)